metrics/interface/src/main/java/org/apache/iotdb/metrics/enums/MetricLevel.java

```diff
@@ -17,16 +17,22 @@
  * under the License.
  */
 
-package org.apache.iotdb.metrics.micrometer;
+package org.apache.iotdb.metrics.enums;
 
-import io.micrometer.core.instrument.Meter;
-import io.micrometer.core.instrument.Tags;
+public enum MetricLevel {
+  all(0),
```

Comment:
Is the `all` level necessary?  It always equals to the lowest level.

Reply:
From my perspective, `all` -> `debug`, `normal` -> `info`, `important` -> `warn`, `core` -> `error`